articles/DeployExistingDjangoProject.md

```diff
@@ -83,10 +104,26 @@ create your virtualenv).
 ### Enter your virtualenv name
 
 Once that's done, **enter the name of your virtualenv** in the Virtualenv
-section on the web tab and click OK.  You can just use its short name
+section on the web tab and click OK.
+
+![virtualenv_web_app](path_to_the_screenshot)
```

Comment:
good idea!  happy to accept screenshots, you can add them to the PR, put them in the **files** folder.  there's an example in *Education.md*